deployment/helpers/solana.ts

```diff
@@ -161,3 +193,69 @@ async function addSignature(tx: Transaction, signer: SolanaSigner, signerPk: Pub
     return await (signer.raw() as NodeWallet).signTransaction(tx);
   }
 }
+
+
+/**
+ * Get the prioritization fee in microlamports from the network using a given policy.
+ * If no prioritization fee is found, return the default fee.
+ * 
+ * @param connection - The connection to the network.
+ * @param lockedWritableAccounts - The locked writable accounts.
+ * @param priorityFeePolicy - The policy to use to get the prioritization fee.
+ * @param defaultFee - The default fee to return if no prioritization fee is found.
+ * @returns The prioritization fee.
+ */
+export async function getPrioritizationFee(connection: Connection, 
+  lockedWritableAccounts: PublicKey[], 
+  priorityFeePolicy: PriorityFeePolicy, 
+  defaultFee: number = DEFAULT_COMPUTE_BUDGET_MICROLAMPORTS): Promise<number> {
+
+  const sortedPrioritizationFeeList = await getSortedPrioritizationFeeList(connection, lockedWritableAccounts);
+  if (sortedPrioritizationFeeList.length === 0) {
+    return defaultFee;
+  }
+
+  return getPrioritizationFeeFromList(sortedPrioritizationFeeList, priorityFeePolicy);
+}
+
+/**
+ * Get the sorted prioritization fee list, in ascending order.
+ * @param connection - The connection to the network.
+ * @param lockedWritableAccounts - The locked writable accounts.
+ * @returns The sorted prioritization fee list.
+ */
+export async function getSortedPrioritizationFeeList(connection: Connection, lockedWritableAccounts: PublicKey[]): Promise<RecentPrioritizationFees[]> {
+  const prioritizationFeeConfig = {
+    lockedWritableAccounts: lockedWritableAccounts,
+  }
+
+  const prioritizationFeeList = await connection.getRecentPrioritizationFees(prioritizationFeeConfig);
+  const nonZeroPrioritizationFeeList = prioritizationFeeList.filter((entry) => entry.prioritizationFee > 0);
```

Comment:
Ah, this is going to skew our measurement :thinking: 
For now, I think it's fine, but in the future we might want something that takes into account the zero priority fees too.
The thing is, many of those are probably jito bundles too, so it's actually a challenge to know which ones you can include in the measurement or not :sweat_smile: